### PR TITLE
Quick cast

### PR DESCRIPTION
**Pull request by vlolteanu**
This patch lets you cast spells with the push of a hotkey, rather than swapping out the readied spell.

Something similar has recently been introduced to D2R.

### CODE REVIEW

File: Source/controls/plrctrls.h [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #pragma once
 // Controller actions implementation
 
+#include "player.h"
```
Comment (glebm) [inline Source/controls/plrctrls.h:4]: Our includes should come after:

1. Standard library
2. Other libraries
2. Ours

**Comment by glebm** [comment]
The C++ bits looks good to me 👍

File: Source/player.h [hunk 2]
```diff
@@ -631,7 +631,7 @@ void ClrPlrPath(Player &player);
 bool PosOkPlayer(const Player &player, Point position);
 void MakePlrPath(Player &player, Point targetPosition, bool endspace);
 void CalcPlrStaff(Player &player);
-void CheckPlrSpell(bool isShiftHeld);
+void CheckPlrSpell(bool isShiftHeld, spell_id spellID = Players[MyPlayerId]._pRSpell, spell_type spellType = Players[MyPlayerId]._pRSplType);
```
Comment (AJenbo) [inline Source/player.h:634]: I did not expect it to be possible to set this complex of a default :o

```suggestion
void CheckPlrSpell(bool isShiftHeld, spell_id spellID = MyPlayer->_pRSpell, spell_type spellType = MyPlayer->_pRSplType);
```

File: Source/controls/plrctrls.h (hunk 1 above)
Comment (vlolteanu) [reply Source/controls/plrctrls.h:4]: I know. It turns out clang-format places them in this order them if you don't add a blank line in between.

**Review by AJenbo — APPROVED** [review]

**Pull request merged**